tests/h/models/group_test.py

```diff
@@ -238,259 +238,6 @@ def test_non_public_group():
     assert not group.is_public
 
 
-class TestGroupACL:
-    def test_auth_client_with_matching_authority_may_read_members(
-        self, group, authz_policy
-    ):
-        group.authority = "weewhack.com"
```

Comment:
All of this weird "weewack.com" stuff is gone in favour of using `f"client_authority:{group.authority}"`